### ISSUE TITLE
[v0.14.0] Can not parse multiline variable in env file

### ISSUE THREAD

**Issue by mizchi** [containerd/nerdctl#719]
TSIA

This bug occurs with GCP credential includes `-----BEGIN PRIVATE KEY-----\n....----END PRIVATE KEY-----\n` to me.

## Reproduce

Put `.env`

```env
SECRET_KEY=-----BEGIN PRIVATE KEY-----\nmy-secret-key----END PRIVATE KEY-----\n
```

Put `docker-compose.yml`

```yml
version: '3.8'
services:
  myapp:
    image: alpine
    env_file:
      - .env
    command: echo key-$SECRET_KEY
```

Run

```
$ lima nerdctl compose up
FATA[0000] unexpected character "-" in variable name near "PRIVATE KEY-----\\nmy-secret-key----END PRIVATE KEY-----\\n\n"
```

## Expected

same with docker's `env_file` behavior.

```bash
$ docker compose up
[+] Running 1/0
 ⠿ Container repro-nerdctl-env-myapp-1  Created         0.0s
Attaching to repro-nerdctl-env-myapp-1
repro-nerdctl-env-myapp-1  | key------BEGIN PRIVATE KEY-----nmy-secret-key----END PRIVATE KEY-----n
```

## Versions

```
$ system_profiler SPSoftwareDataType
Software:

    System Software Overview:

      System Version: macOS 12.0.1 (21A559)
      Kernel Version: Darwin 21.1.0
      Boot Volume: Macintosh HD
      Boot Mode: Normal
      Computer Name: mizchi
      User Name: Kotaro Chikuba (kotaro.chikuba)
      Secure Virtual Memory: Enabled
      System Integrity Protection: Enabled
      Time since boot: 28 days 1:35

$ lima uname -a
Linux lima-default 5.13.0-27-generic #29-Ubuntu SMP Wed Jan 12 17:43:56 UTC 2022 aarch64 aarch64 aarch64 GNU/Linux

$ lima --version
limactl version 0.8.1

$ lima nerdctl --version
nerdctl version 0.14.0
```

**Comment by fahedouch** [comment]
this issue is fixed in `nerdctl version 0.16.0`

limactl version `0.8.1` should wrap `0.16.0` : https://github.com/lima-vm/lima/releases/tag/v0.8.1

plz verify that you are pointing to the right lima

**Comment by fahedouch** [comment]
@mizchi I will close this issue do not hesitate to repoen if you still face problems
